internal/acceptance/ssh_key_helpers.go

```diff
@@ -0,0 +1,36 @@
+package acceptance
+
+import (
+	"fmt"
+
+	"github.com/equinix/terraform-provider-equinix/internal/config"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/packethost/packngo"
+)
+
+func TestAccCheckMetalSSHKeyExists(n string, key *packngo.SSHKey) resource.TestCheckFunc {
```

Comment:
I suspect this file should be named `..._test.go`

Reply:
This is not a test, just a helper function, and we've got one like this for most of the resource types. I'm not sure why.

I would name the file `resource_checkers.go` and put there all the other test helpers which check if resource exists remotely.

Reply:
@displague What about this?